### ISSUE TITLE
How to choose the best prompt

### ISSUE THREAD

**Issue by AlNikolaev** [IDEA-Research/GroundingDINO#35]
Thanks a lot for this amazing repo!!!

During my personal experiments I found that sometimes it is difficult to find the best prompt for some unusual objects and for such objects I tried to describe the object using some properties (size, colors, shape, etc). And sometimes this strategy worked successfully, but sometimes not.

What do you think is it possible to somehow extract the information about the "best" prompt for the object for which we know the bounding box?      
 
Thank you!

**Comment by bjccdsrlcr** [comment]
yeah.... I have this same problem too; Can we find a way or there's some best prompt practices we can learn or know? look forward to your reply
